core/src/trezor/ui/layouts/tt.py

```diff
@@ -248,12 +248,12 @@ async def show_address(
         if is_multisig:
             for i, xpub in enumerate(xpubs):
                 cancel = "Next" if i < len(xpubs) - 1 else "Address"
-                desc = "XPUB #%d" % (i + 1)
-                desc += " (yours)" if i == multisig_index else " (cosigner)"
+                desc_qr = "XPUB #%d" % (i + 1)
```

Comment:
this should probably say `desc_xpub`

Reply:
wow, i'm retarded and also missed the notification, sorry ... fixed in 88aa2b6